src/EnergyPlus/EcoRoofManager.cc

```diff
@@ -241,9 +232,9 @@ namespace EcoRoofManager {
                                         state.dataHeatBalSurf->SurfHGrdExt(SurfNum),
                                         state.dataHeatBalSurf->SurfHAirExt(SurfNum));
         }
-
-        Latm = 1.0 * Sigma * 1.0 * state.dataSurface->Surface(SurfNum).ViewFactorGround * pow_4(state.dataEnvrn->GroundTempKelvin) +
-               1.0 * Sigma * 1.0 * state.dataSurface->Surface(SurfNum).ViewFactorSky * pow_4(state.dataEnvrn->SkyTempKelvin);
+        // Long Wave Radiation (W/m^2)
+        Real64 Latm = 1.0 * Sigma * 1.0 * state.dataSurface->Surface(SurfNum).ViewFactorGround * pow_4(state.dataEnvrn->GroundTempKelvin) +
```

Comment:
I think you're missing a few `1.0` factors here.

Reply:
That quiets the double precision issues we saw on a recent branch!

Reply:
I can't think of a derivation that would wind up in this form. But if it were 1.0 ^ var * 1.0 ^ var, or some similar variation, then that would make more sense. I cant change this without a review of this model. Well, I could, but won't.

Reply:
You could preserve this as a comment and then nuke the 1.0s.